dpdispatcher/submission.py

```
@@ -848,32 +846,32 @@ def handle_unexpected_job_state(self):
                 raise RuntimeError(err_msg)
             self.submit_job()
             if self.job_state != JobStatus.unsubmitted:
                 dlog.info(
-                    f"job:{self.job_hash} re-submit after terminated; new job_id is {self.job_id}"
+                    f"job {self.job_hash} re-submit after terminated; new job_id is {self.job_id}"
                 )
                 time.sleep(0.2)
                 self.get_job_state()
                 dlog.info(
-                    f"job:{self.job_hash} job_id:{self.job_id} after re-submitting; the state now is {repr(self.job_state)}"
+                    f"job {self.job_hash} job_id:{self.job_id} after re-submitting; the state now is {repr(self.job_state)}"
```

Ruff (0.14.8)

Use explicit conversion flag

Replace with conversion flag

(RUF010)